lib/command.js

```diff
@@ -347,7 +348,7 @@ const commands = {
 			statusBar.show("Discarding...", null);
 
 			// discard files
-			return Promise.all([git.clean(root, untrackedFiles), git.checkout(root, trackedFiles)])
+			return Promise.all([git.clean(root, untrackedFiles), git.checkoutFiles(root, trackedFiles)])
```

Comment:
I think they should be seperate. But could you rename `git.checkout` to `git.checkoutBranch` and leave `git.checkoutFiles` as `git.checkoutFiles`